content/docs/user-guide/setup-google-drive-remote.md

```diff
@@ -1,9 +1,79 @@
 # Setup a Google Drive DVC Remote
 
-Follow this guide to setup Google Drive as your <abbr>DVC project</abbr>'s
-[remote storage](/doc/command-reference/remote).
+In this guide we explain the existing ways to setup Google Drive
+[remote storage](/doc/command-reference/remote) for your <abbr>DVC
+projects</abbr>, along with the different benefits each one brings.
 
-## Configure a Google Cloud project
+Note that to start using a GDrive remote, you only need to add it with a valid
+URL (see next section), and use any DVC command that requires it (see
+`dvc pull`, `dvc fetch`, `dvc push`). For example:
+
+```dvc
+$ dvc remote add -d myremote gdrive://root/dvc/dvcstore
+$ dvc add file.dat
+$ dvc push -r myremote
+...
+Go to the following link in your browser:
+
+    https://accounts.google.com/o/oauth2/auth # ... copy this link
+
+Enter verification code: # <- enter resulting token
+```
+
+See the [Authorization](#authorization) section for more details.
+
+## URL format for Google Drive remotes
```

Comment:
do we need to mention that it's Google Drive remotes? We are already in the doc ..

would be better to be explicit here as well - Specifying location ?

Reply:
Good point, updated in 34f8b410.